contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -316,10 +321,48 @@ else if (build && push && dataDirs.size() != 1) {
         }
         if ((!build && push) || (build && !push)) {
             log.warn("DEPRECATED : Creating one build job and separate push job is a deprecated strategy. Instead create"
-                    + "just one job with both build and push set as true and pass a list of cluster urls.");
+                     + "just one job with both build and push set as true and pass a list of cluster urls.");
         }
     }
 
+    private String getMatchingServerSupportedCompressionCodec(int nodeId) {
+        AdminClient adminclient = new AdminClient(clusterUrl.get(0),
```

Comment:
I thought about this issue too but then figured that since we keep a cluster symmetric across all colos, this should be ok. That is, we shouldn't ever be in a situation where ro cluster in one colo is running a server version with compression enabled, while the same cluster in a different colo isn't.